discovery/syncer.go

```diff
@@ -471,7 +474,7 @@ func (g *GossipSyncer) handleSyncingChans() {
 // channelGraphSyncer is the main goroutine responsible for ensuring that we
 // properly channel graph state with the remote peer, and also that we only
 // send them messages which actually pass their defined update horizon.
-func (g *GossipSyncer) channelGraphSyncer() {
+func (g *GossipSyncer) channelGraphSyncer(ctx context.Context) {
```

Comment:
should this context also be passed to `synchronizeChanIDs`? I see inside that method it's creating a new ctx, and same for `sendGossipTimestampRange`.

Reply:
good catch!! will  add a commit to address